ui/ceval/src/view.ts

```diff
@@ -217,7 +227,9 @@ export function renderCeval(ctrl: ParentCtrl): VNode | undefined {
       ]
     : [
         pearl ? h('pearl', [pearl]) : null,
-        h('help', [...engineName(instance), h('br'), trans.noarg('inLocalBrowser')]),
+        h('help', [...engineName(instance), h('br'), 
+        analyzable ? trans.noarg('inLocalBrowser'): trans.noarg('unAnalyzablePosition') 
+        ]),
```

Comment:
Also, please run `yarn run format` in the root directory to auto-format your code.